src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -272,6 +272,16 @@ to operate at the non-zero type;
 extensions will specify how software expresses this authority
 for types not defined above.
 
+Capability encodings may also make the set of <<sec_cap_type>> values
+that may be used to seal a particular capability
+depend on the the permissions granted by that capability.
+For example, it can be a useful space optimization to differentiate
+the <<sec_cap_type>> values
+with which capabilities granting <<x_perm>> may be sealed from those
+with which capabilities not granting <<x_perm>> may be sealed.
+(The <<app_cap_description>> does not avail itself of this option,
+but, for example, the CHERIoT capability encoding does.)
```

Comment:
```suggestion
Capability encodings may also make the set of resulting <<sec_cap_type>> values
depend on the the permissions granted by the input capability.
For example, it can be a useful encoding space optimization to differentiate
the <<sec_cap_type>> values for capabilities granting <<x_perm>> from those not granting <<x_perm>> -- the <<x_perm>> in the capability encoding effectively adds an additional bit to the <<sec_cap_type>> field.
NOTE: The <<app_cap_description>> does not make use of this optimization,
but, for example, the CHERIoT capability encoding does.
```

Maybe something like this? I hope I didn't change the meaning?